### ISSUE TITLE
This hasn't been proposed yet

### ISSUE THREAD

**Issue by stevenvachon** [jasnell/proposal-url#4]
Please bring this up with the other TC39 members.

**Comment by jasnell** [comment]
Yes, I will be :-) I'll be doing another pass at the proposal text then will be getting it on the agenda for the September TC39 meeting.

**Comment by stevenvachon** [comment]
@jasnell any updates?

**Comment by jasnell** [comment]
@mylesborins ... interested in helping bring this to TC39?

**Comment by MylesBorins** [comment]
/cc @domenic who might have opinions.

I could potentially help with this, unsure if there is prior art on something like that. Would likely have to adopt the API exactly as it currently exists... and there would be a hanging question if it should land as a global or within a namespace.

Won't be at July meeting, so earliest I could present a strawperson proposal of this would be Sept

**Comment by domenic** [comment]
The only thing I'd be comfortable with here is if 262 wanted to refer to the URL Standard and require implementations to implement it. No Ecma spec text. I don't think that would be much benefit to the ecosystem personally, but creating a fork of the URL spec in Ecma space would be disastrous, especially so soon after we finally ended the W3C forking.
